tyk-headless/tests/deployment-gw-repset_test.yaml

```diff
@@ -0,0 +1,294 @@
+suite: test headless gateway deployment
+templates:
```

Comment:
Do we need to specify all the templates or just `templates/deployment-gw-repset.yaml` will do?

Reply:
on my local, the test suite was working fine without specifying all the templates but on the CI environment, test cases were failing as follows: https://github.com/TykTechnologies/tyk-helm-chart/actions/runs/3459706419/jobs/5775395782#step:4:34

I solved this issue by adding all templates.

Reply:
okay